airbyte-integrations/connectors/source-datadog/acceptance-test-config.yml

```diff
@@ -5,6 +5,8 @@ acceptance_tests:
   spec:
     tests:
       - spec_path: "source_datadog/spec.yaml"
+        backward_compatibility_tests_config:
+          disable_for_version: "0.4.15" # Set default start and end date for incremental sync
```

Comment:
In theory, you only want to disable the backwards compatibility tests if the new version will not work, but the old one worked. 

If the new version provides defaults and works in a scenario when a new one does not, why would it be broken?

Also, usually if you're skipping backwards compat checks, this should signal that the change is breaking, which this is not.

BUT, your build step in CI is failing on something something date can't be parsed something, so seems relevant.